src/app/applications/applications-menu/applications-menu.component.html

```diff
@@ -35,12 +35,14 @@
 
 
     <div class="mat-divider" *ngIf="applicationsData.favorites?.length > 0 || applicationsData.recents?.length > 0"></div>
```

Comment:
Should thiws still check for recents?  won't it display 2 dividers is recents>0 && favorites==0?

Reply:
Yes indeed double divider. Updated PR